travis/travis_makepot

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
```

Comment:
Can you add copyright?

Reply:
@elicoidal there are none in this repo so far

Reply:
I understand but should not we have them?
This is original creation, same as the README script or any Odoo module.

Reply:
@elicoidal done